src/app/catalog/containers/equipment-item/equipment-item.component.ts

```diff
@@ -30,24 +38,40 @@ export class EquipmentItemComponent implements OnInit {
   ngOnInit(): void {
     this.controller.getCatalog();
 
-    this.controller.getEquipmentItemInfo(
-      this.route.snapshot.params['id']
-    ).subscribe(item => {
-      this.mainPageHeaderService.setPageTitle(item.name);
-      this.equipment = item;
-      this.setPhoto(item);
-      this.cdr.markForCheck();
-    });
+    this.controller
+      .getEquipmentItemInfo(this.route.snapshot.params['id'])
+      .subscribe((item) => {
+        this.mainPageHeaderService.setPageTitle(item.name);
+        this.equipment = item;
+        this.setPhoto(item);
+        this.cdr.markForCheck();
+      });
   }
 
   private setPhoto(equipment: Equipment): void {
-    this.controller.getPhotoById(equipment.photoID).subscribe(res => {
+    this.controller.getPhotoById(equipment.photoID).subscribe((res) => {
       const urlCreator = window.URL || window.webkitURL;
       const url = urlCreator.createObjectURL(res);
 
       if (!this.image) return;
 
       this.image.nativeElement.src = url;
-    })
+    });
+  }
+
+  get cannotDecrease(): boolean {
```

Comment:
Why you use get here?

Reply:
In order not to write restriction logic in the template